api/client/service/opts.go

```diff
@@ -199,14 +198,13 @@ func (m *MountOpt) Set(value string) error {
 			mount.Type = swarm.MountType(strings.ToLower(value))
 		case "source", "name", "src":
 			mount.Source = value
-		case "target", "dst", "dest", "destination", "path":
```

Comment:
https://www.google.com/#q=dst+destination

`src`/`dst` is a common acronym pair, more so that `dest`.